src/BTVNanoCommissioning/helpers/func.py

```diff
@@ -40,7 +40,7 @@ def dump_lumi(events, output):
     pairs = pairs[
         np.lexsort(([pairs[:, i] for i in range(pairs.shape[1] - 1, -1, -1)]))
     ]
-    output["fname"] = processor.list_accumulator([events.metadata["filename"]])
+    output["fname"] = processor.set_accumulator([events.metadata["filename"]])
```

Comment:
What's the difference between `list_accumulatir` and `set_accumulator`?

Reply:
set will not count the info with the same name while list does 😄 (since the single file might split to several chunks which is not necessary to store, just to make the file slimmer)